README.md

```diff
@@ -26,28 +26,35 @@ not shipped with upstream Kubernetes.
 ## Installation
 
 Lokomotive provides the lokoctl CLI tool to manage clusters.
-Check the [installation guide](docs/installer/lokoctl.md) to install it.
+Check the [installation guide](https://kinvolk.io/docs/lokomotive/0.5/installer/lokoctl) to install it.
```

Comment:
Why we can't use relative links BTW? I think it would me MUCH better. Also, if we run in preview, then links will be wrong as well?

Reply:
The README should link to the rendered docs. So this is fine here. Otherwise you'll see the markdown files and Github is nice enough but in the current format of docs, the docs portal will be better.